src/Altinn.Notifications.Core/Enums/AddressType.cs

```diff
@@ -5,5 +5,7 @@
 /// </summary>
 public enum AddressType
 {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
     Email
+#pragma warning restore CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
If we should suppress this warnings in enums, make it identical in every enum.

Reply:
sonarcloud marks it as code smells. We would consider marking it as won't fix in sonarcloud to avoid the supressions as well. What do you think?

Reply:
We suppress warnings so Visual Studio is happy :)